src/Cohesity.Powershell/Scripts/StorageDomain/Set-CohesityStorageDomain.ps1

```diff
@@ -0,0 +1,68 @@
+#### USAGE ####
+#   Set-CohesityStorageDomain -Name newmanStorage
+#   Get-CohesityStorageDomain | Set-CohesityStorageDomain -Name newmanStorage
+#   Get-CohesityStorageDomain -Names newmanStorage | Set-CohesityStorageDomain
```

Comment:
Good stuff thinking about the piping. Let's add concrete examples, for updating the storageDomain, and a way to increase the logical/physical compression and other parameters.